lib/main.dart

```diff
@@ -29,32 +31,37 @@ void main() async {
   final _getPairInfoUseCase = GetPairInfoUseCase(_subGraphRepo);
   final _getSwapInfoUseCase = GetSwapInfoUseCase(_getPairInfoUseCase);
   print("Graph QL CLient initialized}");
-  runApp(GraphQLProvider(
-    client: _gQLClient,
-    child: MultiRepositoryProvider(
-      providers: [
-        RepositoryProvider(create: (context) => _subGraphRepo),
-        RepositoryProvider(
-          create: (context) => MLBRepo(_mlbApi),
-        ),
-        RepositoryProvider(
-          create: (context) => NFLRepo(),
-        ),
-        RepositoryProvider(
-          create: (context) => CoinGeckoRepo(_coinGeckoApi),
-        ),
-        RepositoryProvider(create: (context) => _getPairInfoUseCase),
-        RepositoryProvider(create: (context) => _getSwapInfoUseCase),
-        RepositoryProvider(
-            create: (context) => GetBuyInfoUseCase(_getSwapInfoUseCase)),
-        RepositoryProvider(
-            create: (context) => GetSellInfoUseCase(_getSwapInfoUseCase)),
-        RepositoryProvider(
-            create: (context) => GetPoolInfoUseCase(_getPairInfoUseCase)),
-        RepositoryProvider(
-            create: (context) => GetAllLiquidityInfoUseCase(_subGraphRepo)),
-      ],
-      child: MyApp(),
+  runApp(MultiProvider(
+    providers: [
+      Provider<RedeemBloc>(create: (_) => RedeemBloc()),
+    ],
```

Comment:
If you look at the Buttons.dart file under athlete->components, we provide the bloc there